parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -175,13 +178,19 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(super) type LatestFinalizedHeaderSlot<T: Config> = StorageValue<_, u64, ValueQuery>;
 
+	#[pallet::storage]
+	pub(super) type LatestFinalizedHeaderImportTime<T: Config> = StorageValue<_, u64, ValueQuery>;
+
```

Comment:
would it be better we move all LatestFinalizedHeader*** to a structure and make sure updated atomiclly?

Reply:
@yrong good point! I do this for the execution header: https://github.com/Snowfork/snowbridge/blob/main/parachain/pallets/ethereum-beacon-client/src/lib.rs#L178

I considered doing the same for the finalized header, but back then I kept it separate to simplify the forkless upgrade to Rococco, but since that's no longer a consideration, I agree, let me put them in one storage item.